### DISCUSSION TITLE
Image Zoom

### DISCUSSION THREAD

**Discussion by DataHawker** [fuma-nama/fumadocs#851] [Q&A]
I'm dumb. How do I get the image zoom to work? I just know how to bring in images by (![alt](/image.png) ) I have read the docs like 20 times but I can't see where to make this work. Do I have to update the [slug] page and import it into the DocBody?

**Comment by fuma-nama** [comment]
Add the component to your MDX components prop in docs page.tsx

See https://github.com/fuma-nama/fumadocs/discussions/840#discussioncomment-10599386 for details

**Comment by DataHawker** [comment]
Thanks. I got it to work. It's definitely out of the norm for me. Do you have any good recommendations to add the aviator to the right of the theme selector for a dropdown?

**Comment by Gitstar-OC** [comment]
Add it like this, in your `app/docs/page.tsx` 

```js
import { ImageZoom } from "fumadocs-ui/components/image-zoom";
      <DocsBody>
        <MDX
          components={{
            ...defaultMdxComponents,
            img: props => <ImageZoom {...props as any} />
            }}
        />
      </DocsBody>
```
